client/src/global/components/composed/ThumbnailGrid/styles.js

```diff
@@ -26,20 +26,28 @@ export const Grid = styled.div`
   --list-item-padding: ${({ $grid }) => ($grid ? 0 : "14px")};
   --list-item-border: ${({ $grid }) => ($grid ? "none" : "1px solid")};
   --list-item-margin: ${({ $grid }) => ($grid ? "18px" : 0)};
-  --test-var: blue;
 
   display: grid;
-  grid-template-columns: ${({ $minItemWidth }) =>
-    `repeat(auto-fill, minmax(${$minItemWidth}, 1fr))`};
+  grid-template-columns: ${({ $minItemWidth, $grid }) =>
+    $grid ? `repeat(auto-fit, minmax(${$minItemWidth}, 1fr))` : "1fr"};
+
   padding-top: 30px;
 
   ${listUnstyled}
 
-  ${({ $grid, $empty }) => `
-    ${$grid && gridStyles}
-    ${$empty && emptyStyles}
-    ${$grid && $empty && $grid && `margin: auto;`}
-  `}
+  ${({ $grid, $empty }) => {
+    /* eslint-disable no-nested-ternary */
+    const finalStyles =
+      $grid && $empty
+        ? gridStyles + `margin: auto;`
+        : $grid
+        ? gridStyles
+        : $empty
+        ? emptyStyles
+        : null;
```

Comment:
I find this kind of thing really hard to read. (eslint doesn't like it apparently either. 🙂) Is there another way you could write it? (Not saying the way I did is right.  The logic seems more if/elseif than mine.)